service/service.go

```diff
@@ -281,7 +281,7 @@ func New(config Config) (*Service, error) {
 		frameworkConfig.BackOff = frameworkBackOff
 		frameworkConfig.InitCtxFunc = initCtxFunc
 		frameworkConfig.Logger = config.Logger
-		frameworkConfig.Resources = resources
+		frameworkConfig.ResourceRouter = framework.NewDefaultResourceRouter(resources)
```

Comment:
There is the concept of a resource router and we use the default one the framework provides. It simply always returns the resources given here. Thus nothing changes. Soon we can inspect custom objects and execute different resources based on the custom object being reconciled.